src/server/pfs/cmds/cmds.go

```diff
@@ -1299,18 +1299,6 @@ Tags are a low-level resource and should not be accessed directly by most users.
 	return commands
 }
 
-func parseCommits(args []string) (map[string]string, error) {
```

Comment:
This function was unused in the windows build, so moved it to the unix file (lint error otherwise).

Reply:
:+1: